package.json

```diff
@@ -12,11 +12,9 @@
     "pino.d.ts",
     "bin.js",
     "browser.js",
-    "pretty.js",
-    "usage.txt",
     "test",
     "docs",
-    "example.js",
+    "smoke.js",
```

Comment:
does it need to be included in npm package?

Reply:
Yes , midnight-smoker execute the script inside the package